src/components/Button/Button.scss

```diff
@@ -289,26 +287,12 @@ $block: '.#{variables.$ns}button';
             justify-content: center;
         }
 
-        &_side_start,
-        &_side_end {
-            position: absolute;
-            inset-block-start: 0;
-        }
-
         &_side_start {
-            inset-inline-start: var(--_--icon-position);
-
-            & ~ #{$block}__text {
-                padding-inline-start: var(--_--icon-space);
-            }
+            order: 0;
         }
 
         &_side_end {
-            inset-inline-end: var(--_--icon-position);
-
-            & ~ #{$block}__text {
-                padding-inline-end: var(--_--icon-space);
-            }
+            order: 2;
```

Comment:
Default value for order is `0`, let's use`-1` and `1` then?

Reply:
4490b68a4c8192e11597fcf3a6044d164705149a